sdk/tdf_config.go

```diff
@@ -30,6 +32,8 @@ const kHTTPOk = 200
 
 // KASInfo contains Key Access Server information.
 type KASInfo struct {
+	// Can be empty. If empty, it will inherit the global SDK DialOptions.
```

Comment:
Do we want to have separate dialoptions for KAS or perhaps every service?

Reply:
I could see that making sense. KAS definitely.